packages/react-query/package.json

```diff
@@ -30,39 +30,44 @@
     "build": "rm -rf dist esm && tsc -p tsconfig.build.json --declaration --emitDeclarationOnly --declarationDir dist && rollup -c rollup.config.js",
     "lint": "eslint .",
     "npm:publish": "pnpm publish",
-    "prepack": "pnpm build",
+    "prepack": "pnpm build && pnpm test:tsd",
```

Comment:
Always check the type of the d.ts file built with tsd before `pnpm pack` command to increase the stability of the package.